packages/chrome/src/styled/header/StyledBaseHeaderItem.ts

```diff
@@ -15,17 +15,11 @@ import {
 const COMPONENT_ID = 'chrome.base_header_item';
 
 export interface IStyledBaseHeaderItemProps {
-  /**
-   * Horizontally maximize a flex item in the header to take as much space as possible (i.e. breadcrumb container)
-   **/
+  /** Maximizes the width of a flex item in the header (i.e. breadcrumb container) */
   maxX?: boolean;
-  /**
-   * Vertically maximize the height for a header item (i.e. contains a search input)
-   **/
+  /** Maximizes the height of the item (i.e. contains a search input) */
   maxY?: boolean;
-  /**
-   * Round the border radius for a header item (i.e. user icon)
-   **/
+  /** Rounds the border radius of the item (i.e. user icon) */
```

Comment:
Per guidance from Content Design and https://garden.zendesk.com/components/avatar#how-to-use-it-well, I wonder if we should swap ~user~ with "person". Or does "person icon" just sound a bit weird for dev docs?

Reply:
Do we need the bit inside the parenthesis? For me I'm not sure if it adds value as it seems pretty self-explanatory

Reply:
Nope, good call.